maro/simulator/scenarios/data_center/virtual_machine.py

```diff
@@ -9,19 +9,20 @@ class VirtualMachine:
 
     Args:
         id (int): The VM id.
-        vcpu_cores_requirement (int): The amount of virtual cores requested by VM.
+        cpu_cores_requirement (int): The amount of virtual cores requested by VM.
         memory_requirement (int): The memory requested by VM. The unit is (GBs).
         lifetime (int): The lifetime of the VM, that is, deletion tick - creation tick + 1.
     """
-    def __init__(self, id: int, vcpu_cores_requirement: int, memory_requirement: int, lifetime: int):
+    def __init__(self, id: int, cpu_cores_requirement: int, memory_requirement: int, lifetime: int):
         # VM Requirement parameters.
         self.id: int = id
-        self.vcpu_cores_requirement: int = vcpu_cores_requirement
+        self.cpu_cores_requirement: int = cpu_cores_requirement
         self.memory_requirement: int = memory_requirement
-        # The VM lifetime which equals to the deletion tick - creation tick.
+        # The VM lifetime which equals to the deletion tick - creation tick + 1.
```

Comment:
Add an example for this, like

For VM with utilization: {0: 0.2, 1: 0.4, 2: 0.3}, the lifetime will be 2 - 0 + 1 = 3.

Since the ``VM`` class does not maintain the ``creation tick`` and the ``deletion tick``. How to get ``lifetime`` through ``creation tick`` and ``deletion tick`` can be commented in the code part of ``reader``